### PR TITLE
add 2.3 announcement blost

### PR DESCRIPTION
**Pull request by wmorgan**

### CODE REVIEW

File: linkerd.io/content/blog/announcing-linkerd-2-3.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+draft: false
+slug: announcing-linkerd-2.3
+tags: [Linkerd]
+---
+
+![Lego stormtroopers struggling with questions about
+identity](/uploads/lego-stormtroopers.jpg)
+
+Today we're very happy to announce the release of [Linkerd
+2.3](https://github.com/linkerd/linkerd2/releases/tag/stable-2.3). This release
```
Comment (siggy) [inline linkerd.io/content/blog/announcing-linkerd-2-3.md]: https://github.com/linkerd/linkerd2/releases/tag/stable-2.3.0

File: linkerd.io/content/blog/announcing-linkerd-2-3.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+year ago: **Can we make secure communication *easier* than insecure
+communication for Kubernetes?**
+
+This isn't just a theoretical question. In a world where [the majority of web
+browsers and web sites use strong encryption and
+authentication](https://transparencyreport.google.com/https/overview?hl=en)
+with zero effort on the part of the user, it's increasingly strange for
+Kubernetes services to communicate without authentication and over plain text.
+Why should surfing for [cat pictures on
+Reddit](https://www.reddit.com/r/dogpics) have more security guarantees than
```
Comment (klingerf) [inline linkerd.io/content/blog/announcing-linkerd-2-3.md]: I really hate to be the one to say this, but... are you trying to troll cat lovers by linking to the `dogpics` subreddit here? Or should this link be https://www.reddit.com/r/catpics/ ?

File: linkerd.io/content/blog/announcing-linkerd-2-3.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+* Since the control plane also runs on the data plane, communication between
+  control plane components is secured in the same way.
+
+All of the above happens without requiring any configuration by you, the user,
+and is enabled by default. This means that as of the 2.3 release, Linkerd now
+gives you encrypted, authenticated communication between all your meshed
+services, with no effort on your part. That's not 100% of the way to zero
+trust, but it's a major step in that direction.
+
+This release is a major step forward in Linkerd's security roadmap, but it's
```
Comment (klingerf) [inline linkerd.io/content/blog/announcing-linkerd-2-3.md]: You're using "major step" in the last sentence of the previous paragraph and the first sentence of this paragraph, and it sounds a bit awkward. Maybe mix it up a little bit?

File: linkerd.io/content/blog/announcing-linkerd-2-3.md (hunk 2 above)
Comment (wmorgan) [reply linkerd.io/content/blog/announcing-linkerd-2-3.md]: Troll

File: linkerd.io/content/blog/announcing-linkerd-2-3.md (hunk 3 above)
Comment (wmorgan) [reply linkerd.io/content/blog/announcing-linkerd-2-3.md]: Good catch

File: linkerd.io/content/blog/announcing-linkerd-2-3.md (hunk 1 above)
Comment (wmorgan) [reply linkerd.io/content/blog/announcing-linkerd-2-3.md]: Thanks!

**Comment by wmorgan** [comment]
CI test failure seems like a spurious 404. That URL works for me

**Pull request merged**